app/javascript/channels/consumer.js

```diff
@@ -0,0 +1,6 @@
+// Action Cable provides the framework to deal with WebSockets in Rails.
```

Comment:
Whats this file about?

Reply:
This is where we can create consumers. These consumers can then be used to subscribe to a channel, where they act as subscribers to the channel.